### PR TITLE
FIX: issue #1037, get-order command error

### PR DESCRIPTION
**Pull request by lanphan**
fix issue https://github.com/c9s/bbgo/issues/1037

### CODE REVIEW

File: pkg/cmd/orders.go [hunk 1]
```diff
@@ -46,9 +46,17 @@ var getOrderCmd = &cobra.Command{
...
+		if err != nil {
+			return fmt.Errorf("can't get the order-id from flags: %w", err)
+		}
+
+		symbol, err := cmd.Flags().GetString("symbol")
 		if err != nil {
 			return fmt.Errorf("can't get the symbol from flags: %w", err)
 		}
+		if symbol == "" {
+			return fmt.Errorf("symbol is not found")
```
Comment (c9s) [inline pkg/cmd/orders.go:58]: use `symbol is required` ?

**Review by c9s — APPROVED** [review]

File: pkg/cmd/orders.go (hunk 1 above)
Comment (lanphan) [reply pkg/cmd/orders.go:58]: yes, error returned from binance before fix is

```
FATAL cannot execute command error=<APIError> code=-1105, msg=Parameter 'symbol' was empty.
```

Comment (c9s) [reply pkg/cmd/orders.go:58]: yeah I mean let's use the error message "symbol is required"

Comment (lanphan) [reply pkg/cmd/orders.go]: symbol is required param in viber, so that I remove that check, both in get-order and list-orders command

**Pull request merged**